src/CodiceFiscale/Subject.php

```diff
@@ -129,7 +136,7 @@ public function setGender($gender)
     /**
      * Get the belfioreCode.
      *
-     * @return Returns the belfioreCode.
+     * @return string Returns the belfioreCode.
```

Comment:
You could remove **Returns** here and use an uppercase **T** for the initial **the**.